EarTrumpet/EarTrumpet.csproj

```diff
@@ -82,7 +82,7 @@
     <Reference Include="System.Xml" />
     <Reference Include="Windows, Version=255.255.255.255, Culture=neutral, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\..\..\Program Files (x86)\Windows Kits\10\UnionMetadata\Windows.winmd</HintPath>
+      <HintPath>$(MSBuildProgramFiles32)\Windows Kits\10\UnionMetadata\Windows.winmd</HintPath>
```

Comment:
The path applies to a wider range of PCs.

Reply:
We went down this path before end one of the devs found that the variable we wanted to use wasn't set on their PC.  We're going to give this a try and see if this works (it looks like it will, we were trying to use an SDK path before). We'll report back.